### DISCUSSION TITLE
Rahul-Reddy-T

### DISCUSSION THREAD

**Discussion by Rahul-Reddy-T** [Cyfrin/foundry-full-course-cu#2615] [General]
unable to bridge zksync test net tokens as of insufficient balance can any one help me to get zksync test net tokens

**Comment by felixoder** [comment]
At first grab some sepolia from faucets and then do that again.

**Comment by khandelwalmoksh** [comment]
website for sepolia eth: chainlist.org/?testnets=true

**Reply by EngrPips** [reply]
Interesting.
